sql/expression_value.h

```diff
@@ -1418,6 +1434,8 @@ struct EmbeddingValueInfo: public ExpressionValueInfoT<ML::distribution<CellValu
         input. */
     EmbeddingValueInfo(const std::vector<std::shared_ptr<ExpressionValueInfo> > & input);
 
+    static EmbeddingValueInfo fromShape(const DimsVector& shape, StorageType storageType = ST_ATOM);
```

Comment:
In a lot of places, we call the constructor with a brace initialisation for the shape array, and its ambiguous.